WcaOnRails/spec/features/edit_user_spec.rb

```diff
@@ -29,7 +29,8 @@ def submit_form
 
     expect(page).to have_text I18n.t('users.errors.unique',
                                      used_name: existing_user.name,
-                                     used_email: existing_user.email)
+                                     used_email: existing_user.email,
+                                     used_edit_path: Rails.application.routes.url_helpers.edit_user_path(existing_user))
```

Comment:
Doesn't simply calling `edit_user_path(existing_user)` work here?

Reply:
Named routes in RSpec? Unfortunately not, unless we include the url_helpers in RSpec.configure globally. There are other places we already do something similar to what I did here, e.g. `dob_form_path`.

Reply:
Too bad :(